sdk/identity/azure-identity/src/main/java/module-info.java

```diff
@@ -12,6 +12,7 @@
     requires org.linguafranca.pwdb.database;
     requires org.linguafranca.pwdb.kdbx;
     requires org.linguafranca.pwdb.kdbx.simple;
+    requires reactor.core;
```

Comment:
This change should be reverted.

Reply:
According to your comment in the new revision amendments have been made.